.github/workflows/osv-scanner-reusable-pr.yml

```diff
@@ -86,6 +96,20 @@ jobs:
             --output=${{ inputs.matrix-property }}new-results.json
             ${{ inputs.scan-args }}
         continue-on-error: true
+      - name: "Export results"
```

Comment:
I think this should be after osv-scanner-reporter step, which does the diff between the old and new results and saves it into `${{ inputs.matrix-property }}${{ inputs.results-file-name }}`. This is what's actually displayed to the user.